- Remove hardcoded optimizations
- Fix .so versioning
- use -fPIC to build shared libs on all archs

Index: scripts/mk_util.py
--- scripts/mk_util.py.orig
+++ scripts/mk_util.py
@@ -2632,7 +2632,6 @@ def mk_config():
             EXAMP_DEBUG_FLAG = '-g'
             CPPFLAGS     = '%s -DZ3DEBUG -D_DEBUG' % CPPFLAGS
         else:
-            CXXFLAGS     = '%s -O3' % CXXFLAGS
             if GPROF:
                 CXXFLAGS     += '-fomit-frame-pointer'
             CPPFLAGS     = '%s -DNDEBUG -D_EXTERNAL_RELEASE' % CPPFLAGS
@@ -2657,7 +2656,7 @@ def mk_config():
             SO_EXT         = '.so'
             SLIBFLAGS      = '-shared'
         elif sysname == 'OpenBSD':
-            SO_EXT         = '.so'
+            SO_EXT         = '.so.${LIBz3_VERSION}'
             SLIBFLAGS      = '-shared'
         elif sysname  == 'SunOS':
             SO_EXT         = '.so'
@@ -2673,7 +2672,7 @@ def mk_config():
             LIB_EXT        = '.lib'
         else:
             raise MKException('Unsupported platform: %s' % sysname)
-        if is64():
+        if true:
             if not sysname.startswith('CYGWIN') and not sysname.startswith('MSYS') and not sysname.startswith('MINGW'):
                 CXXFLAGS     = '%s -fPIC' % CXXFLAGS
         elif not LINUX_X64:
